src/Globe/TileMesh.js

```diff
@@ -159,10 +161,17 @@ TileMesh.prototype.setBBoxZ = function setBBoxZ(min, max) {
         const trans = this.normal.clone().setLength(delta.y);
 
         this.geometry.boundingSphere.radius = Math.sqrt(delta.x * delta.x + this.oSphere.radius * this.oSphere.radius);
+        this.updateGeometricError();
         this.centerSphere = new THREE.Vector3().addVectors(this.oSphere.center, trans);
     }
 };
 
+TileMesh.prototype.updateGeometricError = function updateGeometricError() {
+    // The geometric error is calculated to have a correct texture display.
+    // For the projection of a texture's texel to be less than or equal to one pixel
+    this.geometricError = this.geometry.boundingSphere.radius / SIZE_TEXTURE_TILE;
```

Comment:
Is our `geometricError` the same concept than 3d-tiles' `geometricError`?
It looks very similar, and maybe we should adapt the same semantic.
For 3d-tiles, the definition is: `The geometricError property is a nonnegative number that defines the error, in meters, introduced if this tile is rendered and its children are not`.
Maybe you should explain the logic of the formula used, but IMHO:
- depending on a texture size to compute a geometric error is a bit strange
- on the other hand, why doesn't this depends on the number of segments / vertices of a given node? (a node with more segments can be more precise)

Here's a drawing showing `geometricError` for 3d-tiles (in black) ([source](http://cesiumjs.org/publications.html#rendering-the-whole-wide-world)):
![capture d ecran de 2017-05-04 08-49-51](https://cloud.githubusercontent.com/assets/2198295/25693066/4cb99cb0-30a7-11e7-89a9-8eb9750ce6aa.png)